client/src/pages/email-sign-up.tsx

```diff
@@ -138,21 +131,36 @@ function AcceptPrivacyTerms({
         <title>{t('misc.email-signup')} | freeCodeCamp.org</title>
       </Helmet>
       <Grid>
+        {isSignedIn && completedChallengeCount < 2 ? (
```

Comment:
```suggestion
        {isSignedIn && completedChallengeCount < 1 ? (
```

Is this what you are proposing @naomi-lgbt

Reply:
Yep!